pkg/cluster/cluster.go

```diff
@@ -118,7 +118,7 @@ func (c *Cluster) setup() error {
 		return fmt.Errorf("invalid cluster spec: %v", err)
 	}
 
-	var phase spec.ClusterPhase
+	phase := spec.ClusterPhase("")
```

Comment:
why change tis?

can we define ClusterPhase("") as uninitialized or something if we want to make the change here?